app/packages/operators/src/CustomPanel.tsx

```diff
@@ -55,15 +63,23 @@ export function CustomPanel(props: CustomPanelProps) {
   if (!panelSchema)
     return (
       <CenteredStack spacing={1}>
-        <Typography variant="h4">{panelLabel || "Operator Panel"}</Typography>
-        <Typography color="text.secondary">
-          Operator panel &quot;
-          <Typography component="span">{panelName}</Typography>&quot; is not
-          configured yet.
-        </Typography>
-        <Typography component="pre" color="text.tertiary">
-          {panelId}
-        </Typography>
+        {!onLoadError && (
+          <>
+            <SpinnerContainer>
+              <LoadingSpinner />
+            </SpinnerContainer>
+            <Typography variant="h5">Still loading...</Typography>
+            <Typography
+              variant="body1"
+              color="text.secondary"
+              style={{ textAlign: "center" }}
+            >
+              This panel is taking longer than expected to load.
+              <br />
+              You can continue to work with other panels while this loads.
```

Comment:
```suggestion
              You can continue working in other panels while this loads.
```

Same change (just matched Sejal's design), but this wording does communicate a bit better to not necessarily close this current panel while it loads